packages/desktop_security_center/lib/src/l10n/app_de.arb

```diff
@@ -0,0 +1,37 @@
+{
+    "appTitle": "Desktop Security Center",
+    "ubuntuPro": "Ubuntu Pro",
+    "ubuntuProEnabled": "Ubuntu Pro ist aktiviert",
+    "ubuntuProDisabled": "Nicht geschützt mit Ubuntu Pro",
+    "getUbuntuPro": "Ubuntu Pro erhalten",
+    "attachThisMachine": "Diese Maschine anbinden",
+    "ubuntuProFreeDesc": "Immer kostenlos für den persönlichen Gebrauch.\n30-Tage-Testversion für Unternehmen.",
+    "esmLabel": "Erweiterte Sicherheitswartung (ESM)",
+    "esmDesc": "ESM bietet zusätzliche Sicherheitspatches für 10 Jahre für mehr als 23.000 Open-Source-Pakete. Es ermöglicht ein kontinuierliches Schwachstellenmanagement für kritische, hohe und mittlere CVEs.",
+    "proKernelLivepatchLabel": "Kernel-Livepatch",
+    "proEsmMainPackages": "Hauptpakete (esm-infra)",
+    "proEsmMainPackagesDesc": "Sicherheitsaktualisierungen für 2.300 Ubuntu Main-Pakete bis 2034",
+    "proEsmUniversePackages": "Universe-Pakete (esm-apps)",
+    "proEsmUniversePackagesDesc": "Zusätzliche Sicherheitsaktualisierungen für 23.000 Ubuntu Universe-Pakete bis 2034",
+    "proKernelLivepatchDesc": "Wenden Sie Kernel-Sicherheitsaktualisierungen an, während das System läuft.",
+    "proShowLivepatchStatusInTopBar": "Livepatch-Status in der oberen Leiste anzeigen",
+    "proComplianceAndHardening": "Compliance und Härtung",
+    "proComplianceAndHardeningWarning": "Nur zur Einhaltung von FedRAMP, HIPAA und anderen Compliance- und Härtungsanforderungen empfohlen.",
+    "proUsg": "Ubuntu-Sicherheitsleitfaden (USG)",
+    "proUsgDesc": "Automatisiert das Härten und Auditieren mit CIS-Benchmark- und DISA-STIG-Profilen und ermöglicht gleichzeitig umgebungsspezifische Anpassungen. ",
+    "proUsgDocumentation": "USG-Dokumentation",
+    "proFips": "FIPS 140-2",
+    "proFipsDesc": "Ein von der US-amerikanischen und kanadischen Regierung zertifiziertes kryptografisches Modul, das die Einhaltung des Datenschutzstandards FIPS 140-2 bestätigt. ",
+    "proFipsDocumentation": "FIPS-Dokumentation",
+    "proDisablePro": "Ubuntu Pro deaktivieren",
+    "proDialogMagicAttach": "Magic attach with this code at ",
```

Comment:
Yes this "magic" is not very useful jargon coming from the Ubuntu Pro API. I'd suggest you to simply ignore that word for the translation.

Reply:
Even without “magic” I don't really understand the meaning of the "proDialogMagicAttach" string, which would make my translation a game of chance.

Reply:
Makes sense!